Examples/TicTacToe/tic-tac-toe/Sources/GameSwiftUI/GameView.swift

```diff
@@ -2,6 +2,7 @@ import ComposableArchitecture
 import GameCore
 import SwiftUI
 
+@MainActor
```

Comment:
is this needed or just left over test code?

Reply:
It's needed in 5.10, or we need to annotate the helper methods.